ext/ruby_prof/rp_call_tree.c

```diff
@@ -2,9 +2,10 @@
    Please see the LICENSE file for copyright and distribution information */
 
 #include "rp_call_tree.h"
+#include "rp_call_trees.h"
```

Comment:
This creates a circular reference between prof_call_tree_t and prof_call_trees_t. Can we avoid this?